src/pages/shared/components/SelectGasMenu/SelectGasMenu.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { formatCurrency } from 'utils/formatters';
+
+import { LinkTextSmall } from 'shared/commonStyles';
+import { media } from 'shared/media';
+
+import SelectGasMenuBody from './SelectGasMenuBody';
+
+import { ROUTES } from 'constants/routes';
+import DropdownPanel from 'components/DropdownPanel';
+
+export const SelectGasMenu = ({ gasPrice }: { gasPrice: number }) => {
+	const { t } = useTranslation();
+	const { pathname } = useLocation();
+
+	const [gasDropdownIsOpen, setGasDropdownIsOpen] = useState(false);
+	const setDropdownIsOpen = (isOpen: boolean) => {
+		if (!isOpen && !gasDropdownIsOpen) {
+			return;
+		}
+		setGasDropdownIsOpen(isOpen);
+	};
+
+	return (
+		<StyledDropdownPanel
+			height="auto"
+			isOpen={gasDropdownIsOpen}
+			handleClose={() => setDropdownIsOpen(false)}
+			width="170px"
+			onHeaderClick={() => setDropdownIsOpen(!gasDropdownIsOpen)}
+			header={
+				<GasEditFields>
+					<div>{formatCurrency(gasPrice) || 0}</div>
+					{pathname === ROUTES.Options.CreateMarketModal ||
+					pathname === ROUTES.Assets.Options.CreateMarketModal ? (
+						<PaddedLinkText>{t('common.actions.edit')}</PaddedLinkText>
```

Comment:
This is kinda hacky... make a a default style to this, and override it. You can give a className to this text label, and then do something like 

```
const StyledSelectedGasMenu = styled(SelectedGasMenu)`
  .text-label {
    padding-top: 2px;
  }
`